static/scripts/files-storage.js

```diff
@@ -284,3 +280,65 @@ $(document).ready(() => {
 	}
 	$('.file').on('mouseout', fileMouseOutHandler);
 });
+
+window.videoClick = function videoClick(e) {
+	e.stopPropagation();
+	e.preventDefault();
+};
+
+$('.videoclick').on('click', (e) => {
+	window.videoClick(e);
+});
+
+$('.videostop').on('click', () => {
+	window.videojs('my-video').pause();
+	$('#link').html('');
+	$('#picture').attr('src', '');
+	$('#file-view').css('display', 'none');
+});
+
+$('.videostop').on('keypress', (e) => {
+	if (e.key === 'Enter' || e.key === ' ') {
+		document.activeElement.click();
+	}
+});
+
+window.fileViewer = function fileViewer(type, name, id) {
+	$('#my-video').css('display', 'none');
+	let win;
+	const src = `${apiBasePath}/download/${id}/${name}`;
+	switch (type) {
+		case `image/${type.substr(6)}`:
+			window.location.href = '#file-view';
+			$('#file-view').css('display', '');
+			$('#picture').attr('src', src);
+			$('#picture').attr('alt', $t('files.img_alt.altInfoTheImage', { imgName: name }));
+			$('.videostop').trigger('focus');
+			break;
+		case `audio/${type.substr(6)}`:
+		case `video/${type.substr(6)}`:
+			window.location.href = '#file-view';
+			$('#file-view').css('display', '');
+			window.videojs('my-video').src({ type, src });
+			$('#my-video').css('display', '');
+			$('.videostop').trigger('focus');
+			break;
+		default:
+			$('#file-view').hide();
+			win = window.open(src, '_blank');
+			win.focus();
+	}
+};
+
+$('.fileviewer').on('click', function determineViewer(e) {
+	const fileviewertype = this.getAttribute('data-file-viewer-type');
```

Comment:
vars should be camelCase no?

Reply:
No, the getAttribute method find by exact string. And HTML conversation of data attrs are in case-kebab-style!